docs/notebooks/01-spd-helmholtz.ipynb

```diff
@@ -1111,7 +1114,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's have a look at the solution, remember we can `plot` both meshes and Functions."
+    "Let's have a look at the solution.\n",
+    "The built-in plotting functions correspond with those from matplotlib, like `tripcolor`, `tricontour`, `tricontourf`."
```

Comment:
Link to matplotlib docs? (e.g. https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.tripcolor.html)

Reply:
Done! I used the URL that goes to the current version of matplotlib (which of course will change) rather than specifically 3.1.1.